flink-connector/src/main/java/com/datastrato/gravitino/flink/connector/hive/HivePropertiesConverter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.flink.connector.hive;
+
+import static com.datastrato.gravitino.catalog.hive.HiveCatalogPropertiesMeta.METASTORE_URIS;
+
+import com.datastrato.gravitino.flink.connector.PropertiesConverter;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Maps;
+import java.util.Map;
+import java.util.Set;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.table.catalog.CommonCatalogOptions;
+import org.apache.flink.table.catalog.hive.HiveCatalog;
+import org.apache.flink.table.catalog.hive.factories.HiveCatalogFactoryOptions;
+import org.apache.flink.table.factories.FactoryUtil;
+import org.apache.hadoop.hive.conf.HiveConf;
+
+public class HivePropertiesConverter implements PropertiesConverter {
+
+  private HivePropertiesConverter() {}
+
+  public static final HivePropertiesConverter INSTANCE = new HivePropertiesConverter();
+
+  public static final Map<String, String> HIVE_CONFIG_TO_GRAVITINO =
+      ImmutableMap.of(HiveConf.ConfVars.METASTOREURIS.varname, METASTORE_URIS);
+
+  private static final Set<String> SUPPORTED_OPTIONS =
+      ImmutableSet.of(
+          HiveCatalogFactoryOptions.DEFAULT_DATABASE.key(),
+          HiveCatalogFactoryOptions.HIVE_CONF_DIR.key(),
+          HiveCatalogFactoryOptions.HADOOP_CONF_DIR.key(),
+          HiveCatalogFactoryOptions.HIVE_VERSION.key(),
+          FactoryUtil.PROPERTY_VERSION.key(),
+          HiveCatalogFactoryOptions.HADOOP_CONF_DIR.key(),
+          CommonCatalogOptions.CATALOG_TYPE.key());
+
+  @Override
+  public Map<String, String> toGravitinoCatalogProperties(Configuration flinkConf) {
```

Comment:
could you add `flink.bypass.` PREFIX for the properties stored by flink? to keep consistent with spark and trino

Reply:
Done